src/wms/src/main/java/org/geoserver/wms/legendgraphic/ColorMapLegendCreator.java

```diff
@@ -275,9 +275,10 @@ public void setBackgroundColor(final Color backGroundColor) {
         }
 
         public void setBand(final SelectedChannelType grayChannel) {
-            if (grayChannel != null)
-                this.grayChannelName = grayChannel.getChannelName().evaluate(null, String.class);
-
+            /* XXX FIXME: to savoid a compilation error
```

Comment:
savoid -> avoid. And is this something that has been fixed or something to be fixed? If required for compilation, perhaps just remove?

Reply:
My bad: his shouldn't have been added to the commit, as it is an unrelated issue. I will return it to its original state in a another commit.